packages/cozy-client/examples/partial-index.js

```diff
@@ -0,0 +1,52 @@
+const { createClientInteractive } = require('cozy-client')
+const {
+  schema,
+  createContactsBulk,
+  deleteAllContactsBulk,
+  getContactsByIndexesAttributePartialIndex,
+  getContactsByIndexesAttribute,
+  getContactsTest
+} = require('./contacts')
+
+global.fetch = require('node-fetch') // in the browser we have native fetch
+
+const main = async _args => {
+  const client = await createClientInteractive({
+    scope: ['io.cozy.contacts'],
+    uri: process.env.COZY_URL || 'http://cozy.tools:8080',
```

Comment:
Instead of process.env.COZY_URL, could it comes directly from _args ? Why is args with an underscore ?

Reply:
I actually didn't notice the underscore, but yes it could comes from it